staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/compatibility.go

```diff
@@ -364,6 +364,12 @@ func writeFile(t *testing.T, dir string, gvk schema.GroupVersionKind, suffix, ex
 	}
 }
 
+func deleteFile(t *testing.T, dir string, gvk schema.GroupVersionKind, suffix, extension string) {
+	if err := os.Remove(filepath.Join(dir, makeName(gvk)+suffix+"."+extension)); err != nil {
```

Comment:
File not found errors should probably be tolerated?

Reply:
eh, we only call this if the file exists and we want to remove ... not sure I care